File: 3248-snake-in-matrix/3248-snake-in-matrix.py
class Solution:
    def finalPositionOfSnake(self, n: int, commands: List[str]) -> int:
        ans =0
        for command in commands:
            if(command == "RIGHT"):
                ans +=1
            elif(command == "DOWN"):
                 ans +=n
            elif(command == "UP"):
                 ans -=n
            elif(command == "LEFT"):
                 ans -=1
        
        return ans

File: 3248-snake-in-matrix/3248-snake-in-matrix.cpp
class Solution {
public:
    int finalPositionOfSnake(int n, vector<string>& commands) {
        int ans =0;
        for(string command: commands){
            if(command == "RIGHT") ans +=1;
            else if(command == "DOWN") ans +=n;
            else if(command == "UP") ans -=n;
            else if(command == "LEFT") ans -=1;
        }
        return ans;
    }
};